README.md

```diff
@@ -1 +1,184 @@
+
+This is a [Hugo](https://gohugo.io/) theme implemented as a [module](https://gohugo.io/hugo-modules/).  It can be used to generate pages related to the OCW website as well as OCW course sites.
+## Local development
+
+### Dependencies
+
+- [NodeJS](https://nodejs.org/en/download/)
+- [Yarn](https://classic.yarnpkg.com/en/docs/install/#debian-stable)
+- [Go](https://golang.org/doc/install)
+- [Hugo](https://gohugo.io/getting-started/installing/)
+
+If you're running the site for the first time, or if dependencies have changed, install dependencies with:
+
+`yarn install`
+
+### Pre-load Webpack assets
+
+Additionally, you may want to run:
+
+`npm run build:webpack`
+
+Hugo is so fast that the first time you run the site, the Hugo build will happen much more quickly than the webpack build so it's best to run the build once so that assets are ready for when you start the site.  If you don't see any styling the first time you visit the page, force refresh it.  If that doesn't work, try restarting the site.
+
+### External API's
+
+This site requires running instances of [`ocw-studio`](https://github.com/mitodl/ocw-studio) and [`open-discussions`](https://github.com/mitodl/open-discussions) for some functionality.  Search results are provided by `open-discusisons` and `ocw-studio` provides some content for the home page, such as newly added courses and news items.  If you need to work with this functionality you can either run a local instance of either of these projects, or alternatively point at the RC instances and use a browser extension to temporarily disable CORS.
+
+### CORS
+
+The search page at `/search` uses the `open-discussions` search API to source results.  Running this locally and populating it with results can be tedious, so it's often easier to just point your local website at an already running version of the search API.  In order for this to work properly, you will need to use a browser extension for disabling CORS, like Moesif Origin & CORS Changer:
+
+- [Chrome](https://chrome.google.com/webstore/detail/moesif-origin-cors-change/digfbfaphojjndkpccljibejjbppifbc?hl=en-US#:~:text=Moesif%20Origin%20%26%20CORS%20Changer&text=This%20plugin%20allows%20you%20to%20send%20cross%2Ddomain%20requests.&text=This%20plugin%20allows%20you%20to%20send%20cross%2Ddomain%20requests%20directly,Allow%2DOrigin%20set%20to%20*.)
+- [Firefox](https://addons.mozilla.org/en-US/firefox/addon/moesif-origin-cors-changer1/)
+### Environment variables
+
+For development you'll need to set a few environment variables. The
+`.env.example` file will get you started.  Here are the variables and what they do:
+
+| Variable | Example | Description |
+| --- | --- | --- |
+| `SEARCH_API_URL` | `http://localhost:8063/api/v0/search/` | A URL to an `open-discussions` search API to fetch results from|
+| `OCW_STUDIO_BASE_URL` | `http://localhost:8043/` | A URL of an instance of [`ocw-studio`](https://github.com/mitodl/ocw-studio) to fetch home page content from |
```

Comment:
Why does the the theme have a dependency on `OCW_STUDIO_BASE_URL`? I would expect this would only be used by a site like ocw-www which contains the home page.

Reply:
In the `home.html` template in the `www` theme, the `home_course_cards.html` and `ocw_news.html` partials use the Hugo `getJSON` functionality to directly fetch these items using `OCW_STUDIO_BASE_URL`.  If it's not set, `getJSON` will throw an error during the Hugo build.  I agree that this should not be a dependency in spinning up a `www` site, and they can be ignored by adding `ignoreErrors = ["error-remote-getjson"]` to the `config.toml` file or nesting the calls to `getJSON` in a conditional.

Reply:
This should be fixed.

Reply:
@Ferdi this change means that the site will sometimes be missing information on the frontend depending on whether ocw-studio's API call succeeded. The course cards and news items will appear and disappear seemingly at random

Reply:
@noisecapella I understand. I thought we were going to have a flag that would make it behave differentely in prod vs dev.  The problem I have with it is that a theme cannot depend on an external service.